src/ConnectedMode/Binding/IConfigurationPersister.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SonarLint.VisualStudio.Core.Binding;
```

Comment:
I added the missing copyright header in the commit that fixes up the namespaces.